packages/apps/graph/cwd-extra-migrations/2.3.0.0.5_add_index_to_qualname_height_events.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX events_qualname_height ON events (qualname, height DESC);
```

Comment:
Oh, I've just noticed that my suggestion of `CONCURRENTLY` won't work as a CW-D migration, since CW-D runs these migrations in a transaction, but `CONCURRENTLY` index creations need to be run outside transactions. So here's an alternative suggestion:

```suggestion
CREATE INDEX IF NOT EXISTS events_qualname_height_idx ON events (qualname, height DESC);
```

This will at least allow us to run the `CONCURRENTLY` version separately and just push this new `graph` version on top and turn the index creation into a no-op.

Reply:
Why the use of `_idx` at the end? To keep naming consistency the index would just be `tablename_field`